src/external/unicorn-build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Putting this script in external root doesn't look OK to me. Maybe put it in a scripts folder somewhere? It just feel out of place

Reply:
I think I agree, but I also don't know of a better place to put it. I'm happy with it here for now, until we think of somewhere better.

Reply:
I agree that the positioning of this file is suboptimal. I couldn't figure out a better spot though. :(